ui/components/ui/nickname-popover/nickname-popover.component.js

```diff
@@ -54,8 +48,7 @@ const NicknamePopover = ({
           address={address}
           diameter={36}
           className="nickname-popover__identicon"
-          useTokenDetection={useTokenDetection}
-          tokenList={tokenList}
+          image={tokenList[address.toLowerCase()]?.iconUrl}
```

Comment:
Do we need to do this here anymore? Now that the static token list matches the data structure of the dynamic list, can we just  pass the token list to the identicon?